### PR TITLE
Fixes errors on Travis

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Fixed issues related to a new dependency checker on Travis. Closes ckeditor/ckeditor5#1662.

### CODE REVIEW

File: packages/ckeditor5-dev-tests/bin/test-travis.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```
Comment (jodator) [inline packages/ckeditor5-dev-tests/bin/test-travis.sh]: Please add info why `set -e` - I had to google it so others might as well as we do not code in Linux shell that much :)

File: packages/ckeditor5-dev-tests/bin/test-travis.sh [hunk 2]
```diff
@@ -11,10 +13,9 @@ PACKAGE_NAME=$(node -e "console.log( require( process.cwd() + '/package.json' ).
 # The `.ckeditor5_test_environment` file is created by the "install-dependencies" script.
 CKEDITOR5_TEST_ENVIRONMENT=$(cat ${PACKAGE_ROOT}/.ckeditor5_test_environment)
 
-# Those tasks must be executed from the original package.
+# Linter and dependency checker tasks must be executed from the original package.
 yarn run lint && \
-${ROOT_BIN}/ckeditor5-dev-tests-check-dependencies
-
+${ROOT_BIN}/ckeditor5-dev-tests-check-dependencies && \
```
Comment (jodator) [inline packages/ckeditor5-dev-tests/bin/test-travis.sh]: Taking that we have `set -e` it the `&&` still necessary?

Looks safe but might be superfluous...

File: packages/ckeditor5-dev-tests/bin/test-travis.sh (hunk 1 above)
Comment (pomek) [reply packages/ckeditor5-dev-tests/bin/test-travis.sh]: Done.

File: packages/ckeditor5-dev-tests/bin/test-travis.sh (hunk 2 above)
Comment (pomek) [reply packages/ckeditor5-dev-tests/bin/test-travis.sh]: I added two solutions which fix one problem. Most probably we don't need this change but I am safer if we would keep it.

**Comment by pomek** [comment]
https://travis-ci.org/ckeditor/ckeditor5-autoformat/builds/518178545 – this build fails because our dep-check returns some errors.
https://travis-ci.org/ckeditor/ckeditor5-autoformat/builds/518180711 – after fixing dep-check errors, build works. All tests finish their work correctly.

**Pull request merged**